e2e/components/Tabs/Tabs-test.avt.e2e.js

```diff
@@ -0,0 +1,215 @@
+    });
+    await expect(page).toHaveNoACViolations('Tabs--dismissable-with-icons');
+  });
+
+  test('accessibility-checker tabs icon 20 only', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--icon-20-only',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--icon-20-only');
+  });
+
+  test('accessibility-checker tabs icon only', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--icon-only',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--icon-only');
+  });
+
+  test('accessibility-checker tabs manual', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Slider',
+      id: 'components-tabs--manual',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--manual');
+  });
+
+  test('accessibility-checker tabs skeleton', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--skeleton',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs-skeleton');
+  });
+
+  test('accessibility-checker tabs components tabs with icons', async ({
+    page,
+  }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--with-icons',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--with-icons');
+  });
+
+  test.slow('default state - keyboard nav', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--default',
+      globals: {
+        theme: 'white',
+      },
+    });
+
+    // Focus on the first tab via keyboard navigation
+    await page.keyboard.press('Tab');
+    await expect(page.getByRole('tab', { name: 'Tab label 1' })).toBeVisible();
+
+    // Focus should be on content within the first Tab
+    await page.keyboard.press('Tab');
+    await expect(
+      page.getByRole('tabpanel', { name: 'Tab Panel 1' })
+    ).toBeFocused();
+
+    // Moving back one tab stop we should back on Tab label 1
+    await page.keyboard.press('Shift+Tab');
+    await expect(page.getByRole('tab', { name: 'Tab label 1' })).toBeFocused();
+
+    // Nav through the default tab panel via keyboard navigation
+    await page.keyboard.press('ArrowRight');
+    await expect(page.getByRole('tab', { name: 'Tab label 2' })).toBeVisible();
+
+    // Move through the actions in the second tab
+    await page.keyboard.press('Tab');
+    await expect(page.getByRole('checkbox')).toBeVisible();
+
+    await page.keyboard.press('Tab');
+    await expect(page.getByRole('button', { name: 'Submit' })).toBeVisible();
+
+    await page.keyboard.press('Tab');
+    await expect(page.getByRole('textbox')).toBeVisible();
```

Comment:
I don't think we need to test for focus moving through the tab panel items. Could we add a test instead to make sure that when you arrow to the last tab item, it goes back to the start?